.github/README.md

```diff
@@ -0,0 +1,38 @@
+# Renovate Configuration
+
+## What is Renovate?
+
+[Renovate](https://www.mend.io/free-developer-tools/renovate/) is an open-source dependency manager.
+The `Renovate` Github app is configured to automatically open PRs in an attempt to update configured dependencies of different kinds.
+The configuration is managed in [renovate.json5](./renovate.json5).
+
+## How does Renovate work?
+
+`Renovate` is able to automatically identify dependencies of different kinds in different files and start opening PRs once a basic configuration file is found in the repo and the app is enabled. `Renovate` has a lot of different configuration options beyond the supported dependencies which are well documented [here](https://docs.renovatebot.com/).
+
+However, in cases such as this repo where the dependencies are managed in proprietary files, additional configuration is required.
+
+## How is Renovate used in this repo?
+
+In this repo, we have a variety of dependencies pinned to specific versions, primarly to create reproducible, testable builds.\
```

Comment:
Is the trailing backslash needed here? That practice is discouraged here (though I'm not a Markdown expert by any means):

https://www.markdownguide.org/basic-syntax/